### PR TITLE
B 1.2.4 - Resolver form

### PR DESCRIPTION
**Pull request by mohamedaldahoul**
# Description

Add a few sentences describing the overall goals of the pull request. Anything the reviewer should know? Any gotchas?

# Reproduction

Add some clear steps (bullet points) how the reviewer can reproduce the feature, bug.

# Review pull request

Keep the review short and concise, give sugestions on how to fix potential problems and syntax related issues. Let the developer know you revied their PR.

- [x] Checkout the changed files
- [x] `git fetch`, `git checkout BRANCH_NAME`
- [x] `npm start` and see if it works (manual testing)
- [x] `npm test` to see that all the tests are running

If any of these fails, the PR can not be approved or merged, please leave a comment about this and notify the developer.

### CODE REVIEW

**Comment by Babymac** [comment]
Merge conflict in this branch, please fix these and make a new commit.

File: backend/types/user.js [hunk 1]
```diff
@@ -51,3 +51,25 @@ export const resolvers = {
   },
 };
 
+
+export default (prisma, DataTypes) => { //prisma = sequelize
```
Comment (Babymac) [inline backend/types/user.js]: What is the use of all the comments?

File: backend/types/user.js [hunk 2]
```diff
@@ -51,3 +51,25 @@ export const resolvers = {
...
+  const User = prisma.define('User', { //prisma = sequelize
+    username: {
+      type: DataTypes.STRING,
+    },
+  });
+
+User.associate = (types) => {
+  // 1 to many with board
+  User.hasMany(types.form, { //form = board
+      oreignKey: 'owner',
```
Comment (Babymac) [inline backend/types/user.js]: Is this a typo or is "oreignKey" really right? Not "foreignKey"?

Comment (mohamedaldahoul) [reply backend/types/user.js]: it's mistaken, I was working around the code, and I will edit it, I think I should give it oreingKey, is a user. I will look into it.

File: backend/types/user.js (hunk 1 above)
Comment (mohamedaldahoul) [reply backend/types/user.js]: just moved my mistake with the code I was building on one of the copies

File: backend/types/user.js [hunk 3]
```diff
@@ -37,39 +36,32 @@ export const typeDefs = gql`
...
+    users: () => Promise.resolve(users)
   },
   Mutation: {
     addUser: (root, args) => {
       const user = args.user;
       console.log(user);
       users.push(user);
 
-      await mysql.query("INSERT IN ");
+      // await mysql.query("INSERT IN ");
```
Comment (Babymac) [inline backend/types/user.js]: Why is this code commented out? Is it not in use, and wont be, then remove it.

File: backend/types/user.js [hunk 4]
```diff
@@ -37,39 +36,32 @@ export const typeDefs = gql`
...
+      mysql.query("INSERT IN "); //removed await to be able to commit
```
Comment (Babymac) [inline backend/types/user.js]: "//removed await to be able to commit" What does this mean?

**Pull request merged**